openstudiocore/src/utilities/bcl/templates/EnergyPlusMeasure/measure.rb

```diff
@@ -1,3 +1,5 @@
+# insert your copyright here
```

Comment:
@nllong you can update these to be whatever we want now

Reply:
that text seems fine to me.